products/chromeos.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Chrome OS
+category: os
+tags: linux-distribution
+iconSlug: googlechrome
+permalink: /chromeos
+alternate_urls:
+-   /chrome-os
+versionCommand: cat /etc/issue
+releasePolicyLink: https://chromeos.dev/en/education/chromeos-lts
+changelogTemplate: https://support.google.com/chrome/a/answer/10314655/__RELEASE_CYCLE__
+releaseColumn: false
+activeSupportColumn: false
+releaseDateColumn: true
+
+releases:
```

Comment:
Would be great to have a custom script scraping information in the _Chrome version & Stable channel release date_ on https://support.google.com/chrome/a/answer/10314655.